images/win/scripts/ImageHelpers/ImageHelpers.psm1

```diff
@@ -4,6 +4,7 @@ param()
 . $PSScriptRoot\PathHelpers.ps1
 . $PSScriptRoot\InstallHelpers.ps1
 . $PSScriptRoot\MarkdownHelpers.ps1
+. $PSScriptRoot\InstallChoco.ps1
```

Comment:
I suggest to rename it to `ChocoHelpers.ps1` so this file can be reused for more choco-related helpers in future

Reply:
ok I have updated this file